app/src/main/java/com/kylecorry/trail_sense/astronomy/infrastructure/commands/MeteorShowerAlertCommand.kt

```diff
@@ -40,36 +40,42 @@ class MeteorShowerAlertCommand(private val context: Context) : Command<Coordinat
             return
         }
 
-        val notification = Notify.status(
+        val notification = createNotification(shower)
+
+        Notify.send(context, NOTIFICATION_ID, notification)
+    }
+
+    private fun createNotification(shower: MeteorShowerPeak): Notify.StatusNotification {
+        val notificationTitle = context.getString(R.string.meteor_shower)
```

Comment:
Can you inline these with named parameters? You can keep the separate method / constant for the notification ID, but I prefer how it looks with the parameters rather than local variables